UITests/UITests.swift

```diff
@@ -149,6 +149,13 @@ class UITests: XCTestCase {
         app.buttons["More Info"].tap()
         XCTAssert(app.navigationBars["Volleyball?"].exists)
     }
+    
+    func testPopAViewController() {
+        app.buttons["More Info"].tap()
+        XCTAssert(app.navigationBars["Volleyball?"].exists)
+        app.navigationBars.elementBoundByIndex(0).childrenMatchingType(.Button).elementBoundByIndex(0).tap()
```

Comment:
If I try to do this:

```
app.navigationBars.element.childrenMatchingType(.Button).element.tap()
```

I get the error "Multiple matches found".

Could you describe further what you mean?